apps/pollbook/backend/src/app.test.ts

```diff
@@ -707,3 +716,45 @@ test('change a voter address with various formats', async () => {
     }
   });
 });
+
+test('programCard and unprogramCard', async () => {
+  await withApp(async ({ apiClient, auth: authApi, workspace }) => {
```

Comment:
you will need to rename this to localApilClient after the merge on my branch which is why your test is failing rn.